sql.txt

```diff
@@ -1,14 +1,25 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+SELECT CustomerName, ContactName, Country FROM Customers;
+SELECT Country FROM Customers ORDER BY Country;
+SELECT CustomerName, ContactName, Country FROM Customers LIMIT 10 OFFSET 10;
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM Customers WHERE ContactName LIKE "A%" ; 
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND quantity >"10"; 
 
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+INSERT INTO products VALUES (78,"bread",2,2,"4 boxes",9.99);
+INSERT INTO products VALUES (79,"peanuts",4,9,"14 boxes",16.99);
+INSERT INTO products VALUES (80,"milk",1,2,"3 boxes",5.99);
```

Comment:
This is incorrect because the sql doesn't say which columns to add the values. Sql should be written like this or as one statement.

INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES (78,"bread",2,2,"4 boxes",9.99);
INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES (79,"peanuts",4,9,"14 boxes",16.99);
INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES (80,"milk",1,2,"3 boxes",5.99);